arabic/civilization/civilization.md

```diff
@@ -0,0 +1,9 @@
+تعتبر الحضارة الإسلامية من أعظم الحضارات على وجه الأرض ويظل أثرها قائما حتى يومنا هذا. وقد أنجبت الحضارة الإسلامية الكثير
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.